tests/functional/utils/model_management.py

```diff
@@ -21,21 +21,32 @@
 from pathlib import Path
 
 
-def wait_endpoint_setup(container):
+def minio_condition(container):
+    if "created" in container.status:
```

Comment:
`return "created" in container.status` will be enough, the same for `serving_condition`